pgraph/actions.go

```diff
@@ -614,6 +614,7 @@ func (g *Graph) Worker(v *Vertex) error {
 func (g *Graph) Start(first bool) { // start or continue
 	log.Printf("State: %v -> %v", g.setState(graphStateStarting), g.getState())
 	defer log.Printf("State: %v -> %v", g.setState(graphStateStarted), g.getState())
+	defer g.prometheus.UpdatePgraphStartTime()
```

Comment:
the whole resource graph "pauses" when we change graphs... This happens in the Pause() method below.

When we start up again it happens in here too.

While paused, if a resource (vertex) is no longer needed in the new graph, it will receive an exit and shutdown permanently.